lib/mixins/dropdown.js

```diff
@@ -181,11 +182,10 @@ export default {
             });
         },
         focusItem(idx, items) {
-            items.forEach((el, i) => {
-                if (i === idx) {
-                    el.focus();
-                }
-            });
+            let el = items.find((el, i) => i === idx)
```

Comment:
Maybe it was `indexOf` that I was thinking of.  one of the array functions is not available to all users.